manageiq-operator/deploy/crds/manageiq.org_manageiqs_crd.yaml

```diff
@@ -76,10 +76,10 @@ spec:
                 exclusive with the OIDCClientSecret and OIDCProviderURL if using openid-connect
               type: string
             httpdAuthenticationType:
-              description: 'Type of httpd authentication (default: internal) Options:
-                internal, external, active-directory, saml, openid-connect Note: external,
+              description: 'Type of httpd authentication (default: internal) Note: external,
                 active-directory, and saml require an httpd container with elevated
                 privileges'
+              pattern: /(active-directory|external|internal|openid-connect|saml)/
```

Comment:
Do you need to anchor this with `^` or `\A`, and `$` or `\z`?  Otherwise someone could send in `super-saml`

Reply:
Done.